clang/include/clang/Basic/DiagnosticSemaKinds.td

```diff
@@ -10165,7 +10165,10 @@ def err_conflicting_sycl_kernel_attributes : Error<
   "conflicting attributes applied to a SYCL kernel">;
 def err_conflicting_sycl_function_attributes : Error<
    "%0 attribute conflicts with '%1' attribute">;
-
+def err_sycl_x_y_z_arguments_must_be_one : Error<
+  "%0 X-, Y- and Z- sizes must be 1 when %1 attribute is used with value 0">;
+def err_intel_attribute_argument_is_not_in_range: Error<
+   "%0 attribute must be in range from 0 to 3">;
```

Comment:
```suggestion
   "The value of %0 attribute must be in range from 0 to 3">;
```

Reply:
Actually, I missed this test case at all.

Reply:
As did all of us :)